src/pgrn-custom-scan.c

```diff
@@ -292,6 +310,7 @@ PGrnSetRelPathlistHook(PlannerInfo *root,
 			scanData = palloc(sizeof(PGrnScanIndexData));
 			scanData->scanKeys =
 				palloc(sizeof(ScanKeyData) * list_length(quals));
+			scanData->type = T_CustomScan;
```

Comment:
I arbitrarily set `T_CustomScan` as the `NodeTag`, but since this is not appropriate, I will reconsider `PGrnScanIndexData` itself.